### PR TITLE
Whats new updates for `v3.9.0rc0`

### PR DESCRIPTION
**Pull request by trexfeathers**
### Description

In preparation for the [3.9](https://github.com/SciTools/iris/discussions/5850) release candidate release.

---
[Consult Iris pull request check list]( https://scitools-iris.readthedocs.io/en/latest/developers_guide/contributing_pull_request_checklist.html)

---
Add any of the below labels to trigger actions on this PR:

- https://github.com/SciTools/iris/labels/benchmark_this

### CODE REVIEW

File: docs/src/whatsnew/3.9.rst [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
    :open:
 
-   The highlights for this major/minor release of Iris include:
+   This is a small release to make two important changes available as soon as
+   possible:
 
-   * N/A
+   * The :mod:`iris.experimental.geovista` module.
+   * Removal of fill value collision warnings in NetCDF saving, which
+     significantly improves Iris' stability when parallel processing.
```
Comment (pp-mo) [inline docs/src/whatsnew/3.9.rst:21]: Not sure stability is the right term?
```suggestion
     significantly improves Iris' performance when parallel processing.
```

**Review by pp-mo — CHANGES_REQUESTED** [review]
Tiny question.
Otherwise all good!

File: docs/src/whatsnew/3.9.rst (hunk 1 above)
Comment (trexfeathers) [reply docs/src/whatsnew/3.9.rst:21]: OK I suppose this is still performance, yeah. I was thinking about the unpredictability and this perhaps misled me!

**Comment by trexfeathers** [comment]
✔

**Pull request merged**